### PR TITLE
[86bydfcen][time-picker] a11y improvements

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
Fixed reading of dropdowns by Screen readers.
Removed unnecessary `aria-hidden` props.
Fixed text in some `aria-label`.

I've uploaded translations to crowdin and waiting for them back.

## How has this been tested?
Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Comment by msereniti** [comment]
Did you forget to push texts to Crowdin?

**Comment by ilyabrower** [comment]
> Did you forget to push texts to Crowdin?

No, I didn't. I just ask Sheila to look at the one point from the task. The task is still in progress, don't look it please, I'll request your review today)

**Comment by ilyabrower** [comment]
Still waiting for the texts from the writers, after that I will publish them in the crowdin

**Review by sheila-semrush — CHANGES_REQUESTED** [review]
1. We need to hide both the status message and the AM/PM description from the SR (we tried similar thing with Misha, at least with aria-describedby it didn't have any side effects). To clarify, right now you can select and read both text strings with SR.

2. Left the approved text strings in comments to the code.

File: semcore/time-picker/src/translations/en.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
-  "titleEmpty": "Time input, no time entered",
-  "hours": "Hours input field",
-  "minutes": "Minutes input field",
-  "format": "Time format {meridiem}"
+  "title": "{time} {meridiem}",
+  "titleEmpty": "Select",
+  "hours": "Hours",
+  "minutes": "Minutes",
+  "formatToggler": "Toggle time period",
+  "changedFormatNotice": "Time format changed to {meridiem}"
```
[the remark this answers is not in the document]
Comment (ilyabrower) [reply semcore/time-picker/src/translations/en.json:7]: fixed

File: semcore/time-picker/src/translations/en.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
-  "title": "Time input, entered time is {time} {meridiem}",
-  "titleEmpty": "Time input, no time entered",
-  "hours": "Hours input field",
-  "minutes": "Minutes input field",
-  "format": "Time format {meridiem}"
+  "title": "{time} {meridiem}",
+  "titleEmpty": "Select",
+  "hours": "Hours",
+  "minutes": "Minutes",
+  "formatToggler": "Toggle time period",
```
[the remark this answers is not in the document]
Comment (ilyabrower) [reply semcore/time-picker/src/translations/en.json:6]: fixed

**Comment by ilyabrower** [comment]
> 1. We need to clear the text from the status message (span) initially (before user changes the time period), and 2 seconds after changing the period. (2 seconds is not exact, let's test it) Because now you can get into it with the SR at any time.
> 2. Left the approved text strings in comments to the code.
> 3. Commited docs update into your branch, pull plz :)

Fixed all

**Review by sheila-semrush — APPROVED** [review]
All's well with the status announcement now!
@j-mnizhek can you look at the docs?

**Review by j-mnizhek — APPROVED** [review]

File: semcore/time-picker/src/PickerInput.jsx [hunk 3]
```diff
@@ -231,15 +219,6 @@ class Minutes extends ItemPicker {
       }
     }
     if (event.key === 'Enter') this.submitChanges(event);
-    if (event.key === 'ArrowRight') {
```
Comment (msereniti) [inline semcore/time-picker/src/PickerInput.jsx:234]: Why did you remove right arrow handling but preserved left arrow handling?

Comment (ilyabrower) [reply semcore/time-picker/src/PickerInput.jsx:234]: It didn't work. This code is for "minutes" part and should have only left handle (to the hours) as in hours there is only right handle (to the minutes)

**Review by msereniti — APPROVED** [review]
